src/components/show-products/index.js

```diff
@@ -10,16 +11,52 @@ import {
 import { TimeFrames } from '../../utils/timeFrames';
 import { compareDates } from '../../utils/compareDates';
 import ContentContainer from '../content-container';
+import { doc, setDoc, serverTimestamp } from '@firebase/firestore';
+import { db } from '../../lib/firebase';
+
 import './styles.css';
 
 export const ShowProducts = () => {
   const { products, loading } = useProducts();
   const { push } = useHistory();
   const { storedValue } = useLocalStorage(LOCAL_STORAGE_LIST_TOKEN);
   const timeFrames = TimeFrames;
+  const ONE_DAY = 1;
 
-  const handleOnChange = (productID) => {
+  const handleOnChange = (e, productID) => {
```

Comment:
To keep the whole code homogeneous I think the argument "e"  could be called "event"

Reply:
This is a good idea, Susana ^^^. It could be like a coding standard in the team. For example: [this line](https://github.com/the-collab-lab/tcl-34-smart-shopping-list/blob/be41ec0006349cd5620fef954cb672d590e705ed/src/pages/AddItem/index.js#L39).

Reply:
thanks for the recommendation guys, I already do

Reply:
Done!